README.md

```diff
@@ -1,6 +1,7 @@
 # <img src="https://raw.githubusercontent.com/bareos/bareos/master/webui/public/img/bareos.png" alt="Bareos" />
 
-[![Build Status](https://travis-ci.org/bareos/bareos.png?branch=master)](https://travis-ci.org/bareos/bareos/branches)
+
+[![Build Status](https://travis-ci.org/bareos/bareos.png?branch=master)](https://travis-ci.org/bareos/bareos/branches)<br><a href="https://www.saucelabs.com/"> <img src="webui/public/img/saucelabs_white_badge.svg" alt="Saucelabs"  width="90px"/></a>
```

Comment:
Is this the recommend way to handle this? Not using img src, that refers to an icon provided by sauce Lab? Also, wouldn't it be better to link to our sauce Lab test results?

Reply:
It was only sent to me via email, but I've found it hosted by them. The link will point to our sauce lab results as soon as they go live, I've noted this on my todo list.